deploy-agent/deployd/agent.py

```diff
@@ -112,12 +111,6 @@ def _send_deploy_status_stats(self, deploy_report):
             tags['stage_name'] = self._response.deployGoal.stageName
         if deploy_report.status_code:
             tags['status_code'] = deploy_report.status_code
-        if deploy_report.output_msg: 
-            if check_telefig_unavailable_error(deploy_report.output_msg):
-                tags['error_source'] = DeployErrorSource.TELEFIG
-                tags['error'] = DeployError.TELEFIG_UNAVAILABLE
-            elif deploy_report.output_msg.find("teletraan_config_manager") != -1:
-                tags['error_source'] = DeployErrorSource.TELEFIG
```

Comment:
Why removed these?

Reply:
That check might have a perf issue due to string find and I plan to have a check for telefig in the next PR. However, to deploy this version we will need to remove this check to avoid the issue while running in production.